src/Renderer/ThreeExtended/FirstPersonControls.js

```diff
@@ -161,6 +165,16 @@ class FirstPersonControls extends THREE.EventDispatcher {
         }
     }
 
+    translateY(dt, sign) {
+        this.camera.position.z += sign * this.options.moveSpeed * dt / 1000;
```

Comment:
Strange name `translateY` to add on `z`.
But it is coherent to keep this name?

Reply:
yes, you're right, so I changed method name to 'translateZ'